libcodechecker/libhandlers/store.py

```diff
@@ -9,21 +9,22 @@
 """
 
 import argparse
+import base64
 import functools
 import json
-import multiprocessing
 import os
+from hashlib import sha256
```

Comment:
Imports are sorted on the *package* name: `argparse`, `base64`, `functools`, `hashlib`, `json`, ...